lib/browser.ts

```diff
@@ -747,8 +747,8 @@ export class ProtractorBrowser extends AbstractExtendedWebDriver {
    * the wrapped webdriver directly.
    *
    * @example
-   * browser.get('https://angularjs.org/');
-   * expect(browser.getCurrentUrl()).toBe('https://angularjs.org/');
+   * await browser.get('https://angularjs.org/');
+   * expect(await browser.getCurrentUrl()).toBe('https://angularjs.org/');
```

Comment:
Actually, you don't need to add `await` into expect when using Jasmine because result comparison performs when promise in expect is already resolved.

Reply:
In JavaScript world, yes. But in TypeScript latest type definitions for Jasmine would not allow it... Let me see if I can correct type definitions, before changing this.

Reply:
Ok, thanks for the quick response

Reply:
I've had a look and it seems that the functionality was available, because Protractor patched Jasmine and it relies on the control flow, which is being removed now. Jasmine itself does not support passing `Promise` into `expect()` function.

So you'll have to use the form described in this PR or alternatively new [expectAsync](https://jasmine.github.io/api/3.3/global.html#expectAsync) function.

Reply:
@devoto13 , thanks for the explanation

Reply:
@devoto13 is right, `jasminewd` was removed in `selenium4` branch

Reply:
Yup, `jasminewd` was removed because we are no longer using the control flow (because selenium-webdriver deprecated their control flow). Also if I can get jasmine3.3 to work, we could also write this as:

`expectAsync(browser.getCurrentUrl()).toBe('https://angularjs.org');`

Could be nifty but I think the way that this is written is great.